move/its/sources/its.move

```diff
@@ -436,38 +435,11 @@ public fun burn_as_distributor<T>(
 // === Special Call Receiving
 public fun set_trusted_addresses(
```

Comment:
add remove_trusted_addresses method

Reply:
all the logic exists in set_trusted_address already, changing some stuff around.